<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDairyStatisticsWidget</class>
 <widget class="QWidget" name="CDairyStatisticsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labelTitle">
       <property name="text">
        <string>2019年的日记</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDairyTotal">
       <property name="text">
        <string>（共3项）</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOpenDairy">
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="listViewStatistics"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
